lib/new_relic/agent/instrumentation/active_support.rb

```diff
@@ -21,7 +23,8 @@
   end
 
   executes do
-    ActiveSupport::Notifications.subscribe(/\.active_support$/,
+    event_names = NewRelic::Agent.config[EVENT_NAMES_PARAMETER].map { |n| Regexp.escape(n) }.join('|')
```

Comment:
Is this code block saying we won't subscribe to anything if the `EVENT_NAMES_PARAMETER` array is empty aka someone didn't configure it?

Reply:
Hmm... well, it won't be empty by default. It'll be set equal to the list of keys in the `EVENT_NAME_TO_METHOD_NAME` constant in the subscriber class. However, you've pointed out a good edge case to defend against here. It is possible that a user could manually set it `[]` instead of disabling the instrumentation. I'll add a check to guard against that. Thanks!